app/models/like.rb

```diff
@@ -2,5 +2,9 @@ class Like < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: :author_id
   belongs_to :posts
 
+  def update_likes_counter
+    post.increment!(:likesCounter)
+  end
```

Comment:
- [ ] **[OPTIONAL]** As the previous reviewer suggested, I would recommend you add an callback for this method

  ```
  after_save :update_likes_counter  
  
  private

  def update_likes_counter
    ...
  end
  ```